BedrockBlockingCommandQueue.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
+#include <libstuff/libstuff.h>
+#include "BedrockCommandQueue.h"
+#include "BedrockCommand.h"
```

Comment:
This could be a forward declaration (`class BedrockCommand;`) but NAB.

Reply:
Ah, true!